src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -42,14 +43,16 @@ static int Last2DigitsTo0(int versionBuild)
             _sdkRootPath = sdkRootPath;
             _sdkVersionBand = sdkVersionBand;
 
-            var manifestDirectoryEnvironmentVariable = Environment.GetEnvironmentVariable("DOTNETSDK_WORKLOAD_MANIFEST_ROOT");
+            var manifestDirectory = Path.Combine(_sdkRootPath, "sdk-manifests", _sdkVersionBand);
+
+            var manifestDirectoryEnvironmentVariable = Environment.GetEnvironmentVariable("DOTNETSDK_WORKLOAD_MANIFEST_ROOTS");
             if (!string.IsNullOrEmpty(manifestDirectoryEnvironmentVariable))
             {
-                _manifestDirectory = manifestDirectoryEnvironmentVariable;
+                _manifestDirectories = manifestDirectoryEnvironmentVariable.Split(';').Concat(new[] { manifestDirectory }).ToArray();
```

Comment:
this should use `:` on non-Windows since that's the standard separator used in e.g. `PATH` there.

Reply:
`Environment.PathSeparator`?